src/fragments.js

```diff
@@ -949,9 +959,26 @@
          *
          * @returns {string} - basic text version of the fragment
          */
-         asText: function() {
+        asText: function() {
             return this.value.asText();
-         }
+        },
+
+        /**
+         * Get the first Image in slice.
+         * @returns {object}
+         */
+        getFirstImage: function() {
+            var fragment = this.value;
+            if(fragment instanceof  Global.Prismic.Fragments.Group) {
```

Comment:
Instead of testing with instanceof for various types, maybe we should test for existence of the getFirstImage function?

Something like if (typeof fragment.getFirstImage === "function")